src/Integration/Integration.csproj

```diff
@@ -76,6 +76,10 @@
     <ProjectReference Include="..\ProgressVS\ProgressVS.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="ETW\" />
```

Comment:
Why is this needed?

Reply:
it's not. A leftover from moving file. Removed it.